src/Command/ClearCommand.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCacheBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCacheBundle\Command;
+
+use FOS\HttpCache\CacheInvalidator;
+use FOS\HttpCacheBundle\CacheManager;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * A command to clear the whole cache from the command line.
+ *
+ * @author Alexander Schranz <alexander@sulu.io>
+ */
+class ClearCommand extends BaseInvalidateCommand
+{
+    use PathSanityCheck;
+
+    protected static $defaultName = 'fos:httpcache:clear';
+
+    /**
+     * If no cache manager is specified explicitly, fos_http_cache.cache_manager
+     * is automatically loaded.
+     *
+     * @param CacheManager|null $cacheManager The cache manager to talk to
+     */
+    public function __construct(CacheManager $cacheManager = null)
+    {
+        parent::__construct($cacheManager);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName(static::$defaultName) // BC with 2.8
```

Comment:
this makes me wonder if we really need it - we don't support symfony 2.8 anymore...